apps/devicepassthrough_apps.py

```diff
@@ -12,7 +12,7 @@
 
 app = typer.Typer()
 
-SEVICE_NAME = "fmo-dynamic-device-passthrough-service"
+SEVICE_NAME = "fmo-dpf"
```

Comment:
I think this is a typo, is it?

Reply:
Heh, good catch -- rebase leftovers, will fix it now!